index.d.ts

```diff
@@ -0,0 +1,525 @@
+ */
+export function disableZoom(window?: Electron.BrowserWindow): void;
+
+/**
+ * A timestamp (`Date.now()`) of when your app launched.
+ */
+export const appLaunchTimestamp: number;
+
+/**
+ * It works by writing a file to `app.getPath('userData')` if it doesn't exist and checks that.
+ * That means it will return true the first time you add this check to your app.
+ *
+ * @returns A `boolean` of whether it's the first time your app is launched.
+ */
+export function isFirstAppLaunch(): boolean;
+
+export interface DarkMode {
+	/**
+	 * Whether the macOS dark mode is enabled.
+	 * On Windows and Linux, it's `false`.
+	 */
+	isEnabled: boolean;
+
+	/**
+	 * The `callback` function is called when the macOS dark mode is toggled.
+	 * Returns a function, that when called, unsubscribes the listener.
+	 * Calling it on Window and Linux works, but it just returns a noop function.
+	 */
+	onChange: (callback: () => void) => void;
+}
+
+/**
+ *
+ * @example
+ *
+ * import {darkMode} from 'electron-util';
+ * console.log(darkMode.isEnabled);
+ * //=> false
+ *
+ * darkMode.onChange(() => {
+ * 	console.log(darkMode.isEnabled);
+ * 	//=> true
+ * });
+ */
+export const darkMode: DarkMode;
+
+export interface SetContentSecurityPolicyOptions {
+	/**
+	 * The session to apply the policy to.
+	 *
+	 * Default: [`electron.session.defaultSession`](https://electronjs.org/docs/api/session#sessiondefaultsession)
+	 */
+	session?: Electron.Session;
+}
+
+/**
+ * Set a [Content Security Policy](https://developers.google.com/web/fundamentals/security/csp/) for your app.
+ * Don't forget to [validate the policy](https://csp-evaluator.withgoogle.com) after changes.
+ *
+ * @param policy - You can put rules on separate lines, but lines must end in a semicolon.
+ * @param options
+ *
+ * @example
+ *
+ * import setContentSecurityPolicy from 'electron-util';
+ *
+ * setContentSecurityPolicy(`
+ * 	default-src 'none';
+ * 	script-src 'self';
+ * 	img-src 'self' data:;
+ * 	style-src 'self';
+ * 	font-src 'self';
+ * 	connect-src 'self' https://api.example.com;
+ * 	base-uri 'none';
+ * 	form-action 'none';
+ * 	frame-ancestors 'none';
+ * `);
+ */
+export function setContentSecurityPolicy(policy: string, options?: SetContentSecurityPolicyOptions): void;
+
+/**
+ * Opens the new issue view on the given GitHub repo in the browser.
+ * Optionally, with some fields like title and body prefilled.
+ *
+ * @param options - The options are passed to the [`new-github-issue-url`](https://github.com/sindresorhus/new-github-issue-url#options) package.
+ *
+ * @example
+ *
+ * import {openNewGitHubIssue} from 'electron-util';
+ *
+ * openNewGitHubIssue({
+ * 	user: 'sindresorhus',
+ * 	repo: 'playground',
+ * 	body: 'Hello'
+ * });
+ */
+// TODO options type?
+export function openNewGitHubIssue(options: unknown): void;
+
+export interface OpenUrlMenuItemOptions extends Electron.MenuItemConstructorOptions {
+	/**
+	 * TODO
+	 */
+	url: string;
```

Comment:
```suggestion
	url?: string;
```

Reply:
Here the `url` option is required: https://github.com/sindresorhus/electron-util/blob/e2ea4fd/index.js#L221-L223